.github/workflows/go.yml

```diff
@@ -4,119 +4,113 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
-    timeout-minutes: 5
```

Comment:
Is there a reason for this timeout increase?

Reply:
It is taking around 7 minutes to run the unit tests with the added flags, so it was getting timed out at 5 minutes. That is why it is increased